### ISSUE TITLE
Two improvements to the Xenium notebook

### ISSUE THREAD

**Issue by LucaMarconato** [scverse/squidpy_notebooks#128]
CC @giovp @LLehner 

The 10x team reviewed the Xenium notebook and found it very useful! (Btw, it's ready to merge https://github.com/scverse/squidpy_notebooks/pull/125).

They suggested to consider two improvements.
- [ ] To show an example of how to use the multimodal segmentation as part of the analysis. The information is readily available in the `SpatialData` object, [as shown here](https://spatialdata.scverse.org/en/latest/tutorials/notebooks/notebooks/examples/technology_xenium.html#exploring-the-multimodal-segmentation).

**Comment by LucaMarconato** [comment]
- [ ] They suggested also to consider using a different dataset for the example since we are using a preview one. I'd suggest to wait until a 5K Xenium Onboard Analysis 3.0 is available (I'll update the notebook in SpatialData when it's available and put the link here: https://github.com/scverse/spatialdata-notebooks/issues/110).

**Comment by LucaMarconato** [comment]
I'd suggest to make a PR for the first issue and then transfer the second task item in a new issue to address later on.

**Comment by giovp** [comment]
update: #125 merged

**Comment by LucaMarconato** [comment]
Yes, I'd keep this to track the usage on the 5k dataset; but also, as pointed out by 10x, it would be nice to plot/use the new multimodal segmentation.
